packages/insomnia-app/webpack/webpack.config.development.babel.js

```diff
@@ -34,6 +37,7 @@ module.exports = {
       __DEV__: true,
       'process.env.NODE_ENV': JSON.stringify('development'),
       'process.env.INSOMNIA_ENV': JSON.stringify('development'),
+      'process.env.releaseDate': JSON.stringify(date),
```

Comment:
Potentially update this to follow the same pattern as the other environment variables 🤗 

```suggestion
      'process.env.RELEASE_DATE': JSON.stringify(date),
```